schematic_api/Dockerfile

```diff
@@ -73,7 +73,9 @@ WORKDIR ${APP_DIR}
 
 # copy other files to app/app
 # Note: run_api.py is not needed
-COPY ./pyproject.toml ./poetry.lock ./config.yml ./main.py ./
+
```

Comment:
fixed the issue related to docker file after configuration class refactor. This part was borrowed from PR: https://github.com/Sage-Bionetworks/schematic/pull/1269